docs: update authd availability for Ubuntu 26.04 LTS archive (#1373)
ubuntu/authd: docs/howto/install-authd.md

## System requirements

* Distribution: Ubuntu Desktop 24.04 LTS or later, or Ubuntu Server 24.04 LTS or later
* Architectures: amd64, arm64

## Install authd

> **Ubuntu 26.04 LTS (Resolute) and later:** `authd` is available directly from the Ubuntu archive.

Reviewer: ```suggestion
On Ubuntu 26.04 LTS, `authd` is available directly from the Ubuntu archive.
```

I don't think we need the "and later" here? What is "later" currently?

I also don't think we need the additional formatting or the release name.

Author: Should I remove the and later from the `SECURITY.md` file too?

```
We provide security updates for the latest version of `authd` on each supported Ubuntu LTS release.

**Ubuntu 26.04 LTS and later**: `authd` is available directly from the Ubuntu archive.

**Ubuntu 24.04 LTS and earlier supported LTS releases**: `authd` is not in the Ubuntu archive and must be installed using the stable PPA.
```

One could argue that for upcoming releases, they will be in archive, so we won't have to update it for each LTS release, unless we wish to make it more explicit. What do you think?

> **Ubuntu 24.04 LTS (Noble):** `authd` must be installed from the [stable PPA](https://launchpad.net/~ubuntu-enterprise-desktop/+archive/ubuntu/authd):

Reviewer 2: Ok, then nevermind
